README.md

```diff
@@ -1 +1,21 @@
 # LFP Appeals Processor API
+
+## How to run
+### Run as a Spring Boot application
+`mvn clean install spring-boot:run`
+
+## Useful Endpoints
+### Health
+http://localhost:9000/actuator/health
+
+### API Documentation (Swagger)
+http://localhost:9000/swagger-ui.html
+
+## Technologies
+- [OpenJDK 11](https://jdk.java.net/archive/)
+- Maven 3.x.x
```

Comment:
I'd suggest removing versions from here. It should not matter much for the reader especially that sometimes differences between major versions are not significant to bother. Having these numbers here will increase maintainability overhead when we bump tools while I'd love to make it as much low maintenance as possible.

Reply:
ok, that's fair enough. I'll leave the java version for now as this one is important

Reply:
I forgot to mention that links, like in case of JDK, are good idea.